m4_sessions/1708-inning/group_one.md

```diff
@@ -9,17 +9,30 @@
 * History, now, and future of transhumanism
 * Discussion humanism and see how people's feelings and thoughts change after the session.
 
-### Session Outline:
-* Whiteboard spectrum of how people feel about this.
-  - Have everyone go one by one, introduce themself and point out where they are in the spectrum.
-* Introduction to material
-  - History, current
-* small group discussions
-  - After we have groups discuss a main point or two, we have another pulse check on what people thing of transhumanism.
-* Future of transhumanism
-  - What role do we as developers play
-* another small group discussion
-  - last pulse check and see where people are in how they feel about this
+### Outline
+
+#### Intro (15 mins)
+	1) Quick overview on Transhumanism and what we will be doing today.
+	2) Turn and talk. What are your thoughts on the matter?
+	3) Divide group into spectrums (move around room) Joe plays devils advocate, against. Spectrums  = for, against, unsure
```

Comment:
Really like this approach.  Be sure each of you can confidently share where you land on the spectrum and why.